docs/products/custom_domain.md

```diff
@@ -67,7 +67,7 @@ The endpoint URL will need to be added to a CNAME record for the domain in quest
 Team leads should be able to help get access to the required resources.
 This guide does not cover the steps required to configure the CNAME record in the DNS configuration.
 
-## Configuring the operator installation.
+<h5><b>Configuring the operator installation</b></h5>
```

Comment:
Why this change? Adding the h5 puts this item under domain certificate which it does not belong to. Why would you want this section not linkable?

Reply:
As agreed, I've moved away from html addition, this is a miss.